moped-database/metadata/tables.yaml

```diff
@@ -3849,53 +2636,49 @@
       - project_priority
       - project_sponsor
       - project_uuid
-      - project_website
       - purchase_order_number
+      - project_website
       - start_date
       - status_id
-      - task_order
```

Comment:
@frankhereford it's kind of hard to assess the changes  to `tables.yaml` to in the github UI, but an noticing `task_order` permissions are being revoked. could this be an oversight?

Reply:
@johnclary I am very grateful that you caught that! 🦅 👁️  The way the file get shuffled around when it's written out by `graphql-engine` makes reading diffs really tricky, I agree.

You are exactly right that I had inadvertently revoked access to `task_order` as well as `knack_project_id` for the moped_projects table. I went and double checked all the other tables to see if I had done it in other places, and I did find that the same thing had happened to `fund` and `dept_unit` in the `moped_proj_funding` tables.

TBH, I am not sure exactly how this happened, but I definitely did it here in [this commit](https://github.com/cityofaustin/atd-moped/pull/539/commits/1cd53ddeebdfbf3391424b3cb1b1cbf8c9e16624). The best I can think of is that I made a mistake and a `graphql-engine` daemon running while I was jumping branches with `git`.  Thanks @mateoclarke for brainstorming on this with me, too.

I've pushed up two commits that restore these columns' permissions. Thank you again!